### PR TITLE
features math formulae, initial commit

### PR DESCRIPTION
**Pull request by friskluft**

### CODE REVIEW

File: docs/f_fractal_dimension.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Fractal dimension features
+
+Suppose $A$ is a shape's area and $P$ is its perimeter, and we are determining $D$.
+
+## FRACT_DIM_BOXCOUNT
+
+The underlying relation is $n = b^{-D}$. The feature is calculated as the slope $-D$ of the plot $\text {log} \: n$ against $\text {log} \: b$.
```
Comment (JesseMckinzie) [inline docs/f_fractal_dimension.md:7]: It may be helpful to add what the variable b represents

**Review by JesseMckinzie — APPROVED** [review]

File: docs/f_2dmoments.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+NORM_CENTRAL_MOMENT_02 $=\eta _{{02}}$\
+NORM_CENTRAL_MOMENT_03 $=\eta _{{03}}$\
+NORM_CENTRAL_MOMENT_11 $=\eta _{{11}}$\
+NORM_CENTRAL_MOMENT_12 $=\eta _{{12}}$\
+NORM_CENTRAL_MOMENT_20 $=\eta _{{20}}$\
+NORM_CENTRAL_MOMENT_21 $=\eta _{{21}}$\
+NORM_CENTRAL_MOMENT_30 $=\eta _{{30}}$
+
+## Hu moments
+Hu invariants HU_M1 through HU_M7 are calculated as\
```
[the remark this answers is not in the document]
Comment (friskluft) [reply docs/f_2dmoments.md:67]: Fixed

File: docs/f_2dmoments.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+HU_M1 $=\eta _{{20}}+\eta _{{02}}$\
+HU_M2 $=(\eta _{{20}}-\eta _{{02}})^{2}+4\eta _{{11}}^{2}$\
+HU_M3 $=(\eta _{{30}}-3\eta _{{12}})^{2}+(3\eta _{{21}}-\eta _{{03}})^{2}$\
+HU_M4 $=(\eta _{{30}}+\eta _{{12}})^{2}+(\eta _{{21}}+\eta _{{03}})^{2}$\
+HU_M5 $=(\eta _{{30}}-3\eta _{{12}})(\eta _{{30}}+\eta _{{12}})[(\eta _{{30}}+\eta _{{12}})^{2}-3(\eta _{{21}}+\eta _{{03}})^{2}]+(3\eta _{{21}}-\eta _{{03}})(\eta _{{21}}+\eta _{{03}})[3(\eta _{{30}}+\eta _{{12}})^{2}-(\eta _{{21}}+\eta _{{03}})^{2}]$\
+HU_M6 $=(\eta _{{20}}-\eta _{{02}})[(\eta _{{30}}+\eta _{{12}})^{2}-(\eta _{{21}}+\eta _{{03}})^{2}]+4\eta _{{11}}(\eta _{{30}}+\eta _{{12}})(\eta _{{21}}+\eta _{{03}})$\
+HU_M7 $=(3\eta _{{21}}-\eta _{{03}})(\eta _{{30}}+\eta _{{12}})[(\eta _{{30}}+\eta _{{12}})^{2}-3(\eta _{{21}}+\eta _{{03}})^{2}]-(\eta _{{30}}-3\eta _{{12}})(\eta _{{21}}+\eta _{{03}})[3(\eta _{{30}}+\eta _{{12}})^{2}-(\eta _{{21}}+\eta _{{03}})^{2}]$\
```
[the remark this answers is not in the document]
Comment (friskluft) [reply docs/f_2dmoments.md:75]: Fixed

File: docs/f_erosion.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+These features use the morphological erosion of the binary ROI image with respect to the $3 \times 3$ structuring element ker. The input ROI image is in the form of an image matrix.
+
+## EROSIONS_2_VANISH
+
+This feature calulates the number of erosion operations on a ROI to vanish in its axis aligned bounding box.
+
+## EROSIONS_2_VANISH_COMPLEMENT
+
+Unlike EROSIONS_2_VANISH which is measured with respect to a ROI's axis aligned bounding box, his feature calculates the number of erosion operations on a ROI to vanish in its convex hull.
```
[the remark this answers is not in the document]
Comment (friskluft) [reply docs/f_erosion.md:11]: Details added

File: docs/f_fractal_dimension.md (hunk 1 above)
Comment (friskluft) [reply docs/f_fractal_dimension.md:7]: Details and math added

**Pull request merged**